### PR TITLE
Refactor: move cncd/{logging,pubsub,queue}/ to server/{logging,pubsub,queue}/

### PR DESCRIPTION
**Pull request by cognifloyd**

### CODE REVIEW

File: server/logging/README [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Go package provides a common interface for storing and streaming logs.
```
[the remark this answers is not in the document]
Comment (cognifloyd) [reply server/logging/README:1]: I just moved them and adjusted the path.
It looks like the 3 packages originated here:
https://github.com/cncd/queue
https://github.com/cncd/pubsub
https://github.com/cncd/logging

[the remark this answers is not in the document]
Comment (cognifloyd) [reply server/logging/README:1]: We could just delete it though. 🤷

[the remark this answers is not in the document]
Comment (cognifloyd) [reply server/logging/README]: I turned them into README.md files. wdyt?

File: server/pubsub/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## History
+
+This was originally published in: https://github.com/cncd/pubsub
+Then it was included in: https://github.com/drone-ci/drone/cncd/pubsub
+And now it is: https://github.com/woodpecker-ci/woodpecker/server/pubsub
+
+## Documentation:
+
+    http://godoc.org/github.com/woodpecker-ci/woodpecker/server/pubsub
```
Comment (anbraten) [inline server/pubsub/README.md:13]:
```suggestion
https://godoc.org/github.com/woodpecker-ci/woodpecker/server/pubsub
```

File: server/pubsub/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# pubusb package
+
+Go package provides a common interface for publish-subscriber messaging.
+
+## History
+
+This was originally published in: https://github.com/cncd/pubsub
+Then it was included in: https://github.com/drone-ci/drone/cncd/pubsub
+And now it is: https://github.com/woodpecker-ci/woodpecker/server/pubsub
```
Comment (anbraten) [inline server/pubsub/README.md:9]:
```suggestion
```

File: server/queue/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# queue package
+
+Go package provides a common interface for working with task queues.
+
+## History
+
+This was originally published in: https://github.com/cncd/queue
+Then it was included in: https://github.com/drone-ci/drone/cncd/queue
+And now it is: https://github.com/woodpecker-ci/woodpecker/server/queue
```
Comment (anbraten) [inline server/queue/README.md:9]:
```suggestion
```

File: server/queue/README.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+## Documentation:
+
+    http://godoc.org/github.com/woodpecker-ci/woodpecker/server/queue
```
Comment (anbraten) [inline server/queue/README.md:13]:
```suggestion
https://godoc.org/github.com/woodpecker-ci/woodpecker/server/queue
```

File: server/logging/README.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+## History
+
+This was originally published in: https://github.com/cncd/logging
+Then it was included in: https://github.com/drone-ci/drone/cncd/logging
+And now it is: https://github.com/woodpecker-ci/woodpecker/server/logging
+
+## Documentation:
+
+    http://godoc.org/github.com/woodpecker-ci/woodpecker/server/logging
```
Comment (anbraten) [inline server/logging/README.md:13]:
```suggestion
https://godoc.org/github.com/woodpecker-ci/woodpecker/server/logging
```

File: server/logging/README.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# logging package
+
+Go package provides a common interface for storing and streaming logs.
+
+## History
+
+This was originally published in: https://github.com/cncd/logging
+Then it was included in: https://github.com/drone-ci/drone/cncd/logging
+And now it is: https://github.com/woodpecker-ci/woodpecker/server/logging
```
Comment (anbraten) [inline server/logging/README.md:9]:
```suggestion
```

File: server/logging/README (hunk 1 above)
[the remark this answers is not in the document]
Comment (anbraten) [reply server/logging/README]: That's nice. Just some small suggestion.

**Review by anbraten — APPROVED** [review]
thanks

**Pull request merged**